cime/config/e3sm/config_grids.xml

```diff
@@ -2499,6 +2557,40 @@
       <map name="LND2ATM_SMAPNAME">cpl/gridmaps/fv0.23x0.31/map_fv0.23x0.31_to_ne240np4_aave_110428.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne512np4" ocn_grid="oRRS15to5">
+      <map name="ATM2OCN_FMAPNAME">cpl/gridmaps/ne512np4/map_ne512np4_to_oRRS15to5_mono.190402.nc</map>
+      <map name="ATM2OCN_SMAPNAME">cpl/gridmaps/ne512np4/map_ne512np4_to_oRRS15to5_mono.190402.nc</map>
+      <map name="ATM2OCN_VMAPNAME">cpl/gridmaps/ne512np4/map_ne512np4_to_oRRS15to5_mono.190402.nc</map>
+      <map name="OCN2ATM_FMAPNAME">cpl/gridmaps/oRRS15to5/map_oRRS15to5_to_ne512np4_mono.190402.nc</map>
+      <map name="OCN2ATM_SMAPNAME">cpl/gridmaps/oRRS15to5/map_oRRS15to5_to_ne512np4_mono.190402.nc</map>
+    </gridmap>
+
+    <gridmap atm_grid="ne512np4" lnd_grid="360x720cru">
+      <map name="ATM2LND_FMAPNAME">cpl/gridmaps/ne512np4/map_ne512np4_to_360x720cru_mono.190409.nc</map>
+      <map name="ATM2LND_SMAPNAME">cpl/gridmaps/ne512np4/map_ne512np4_to_360x720cru_mono.190409.nc</map>
+      <map name="LND2ATM_FMAPNAME">cpl/gridmaps/360x720cru/map_360x720cru_to_ne512np4_mono.190409.nc</map>
+      <map name="LND2ATM_SMAPNAME">cpl/gridmaps/360x720cru/map_360x720cru_to_ne512np4_mono.190409.nc</map>
+    </gridmap>
+
+    <gridmap atm_grid="ne1024np4" ocn_grid="oRRS15to5">
+      <map name="ATM2OCN_FMAPNAME">cpl/gridmaps/ne1024np4/map_ne1024np4_to_oRRS15to5_mono.190509.nc</map>
+      <map name="ATM2OCN_SMAPNAME">cpl/gridmaps/ne1024np4/map_ne1024np4_to_oRRS15to5_mono.190509.nc</map>
+      <map name="ATM2OCN_VMAPNAME">cpl/gridmaps/ne1024np4/map_ne1024np4_to_oRRS15to5_mono.190509.nc</map>
+      <!--map name="OCN2ATM_FMAPNAME">cpl/gridmaps/oRRS15to5/map_oRRS15to5_to_ne1024np4_monotr.190509.nc</map-->
```

Comment:
What's the difference between mono and monotr files? If you want to keep these comments with monotr files, would you want to add a comment on why?

Reply:
Yeah, again this was related to problems with domain checks. "monotr" from ocean to atmos would be the transpose map computed based on the atmos to ocean mono map, while "mono" from ocean to atmos would be a separate independently calculated mono map. Not sure of the details, but again this was a change I implemented when trying to figure out the domain check errors I was running into. I will submit another test to confirm that this was a problem. The monotr map I think is the "preferred" algorithm if it works, but I recall having trouble with it.

Reply:
Mark has a page explaining mapping suffixes https://acme-climate.atlassian.net/wiki/spaces/ED/pages/178848194/Transition+to+TempestRemap+for+Atmosphere+grids .

Reply:
Yeah, I'm familiar with that document. The reason for using mono vs monotr here was that I was getting domain check failures with monotr, but I can revisit this.

Reply:
I just removed the commented out lines. Since the domain files use the mono maps, I think I'd have to replace both domain files and mapping files to test this. Since it's working now, I elected to just remove the commented out ones and move on.